docs/GettingStarted/ExampleProfile1.json

```diff
@@ -25,7 +25,7 @@
         {
           "field": "Column 2",
           "is": "ofType",
-          "value": "numeric"
+          "value": "integer"
         }
       ]
     }
```

Comment:
would it be worth adding an example decimal constraint as well?

Reply:
I think simplicity is key on this example